src/renderers/WebGLRenderer.js

```diff
@@ -1461,6 +1464,8 @@ class WebGLRenderer {
 
 			_this.clear();
```

Comment:
After reading the code again I _think_ `_this.clear();` can be removed now. Otherwise `clear()` is effectively called twice.

Reply:
The previous approach only honored the clear color set via `setClearColor()`. But if you trigger the clear via `background.render( scene )`, then both clear color and `Scene.background = new Color()` are honored.

Reply:
I don't think it's necessarily the case? Again the rationale behind the current transmission clearing logic isn't super clear to me. If `renderBackground` is set to false then this call to "clear" will still be run, for example. I guess it can at least go in an "else" branch?

Reply:
> If renderBackground is set to false then this call to "clear" will still be run, for example. I guess it can at least go in an "else" branch?

That makes sense. This particular clear should be retained in XR.